index.html

```diff
@@ -0,0 +1,270 @@
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="css/flexboxgrid.min.css">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+    integrity="sha384-xxzQGERXS00kBmZW/6qxqJPyxW3UR0BPsL4c8ILaIWXva5kFi7TxkIIaMiKtqV1Q" crossorigin="anonymous">
+  <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600;700&family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
+  <meta charset="UTF-8">
+  <title>youtube</title>
+  <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="css/flexboxgrid.min.css">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css"
+    integrity="sha384-xxzQGERXS00kBmZW/6qxqJPyxW3UR0BPsL4c8ILaIWXva5kFi7TxkIIaMiKtqV1Q" crossorigin="anonymous">
+  <link rel="shortcut icon" type="image/x-icon" href="img/favicon.ico">
+</head>
+
+<body>
+  <nav></nav>
+  <header class="row">
+    <div class="col-md-3 logo-nav">
+      <a href="">
+        <i class="fas fa-bars"></i>
+      </a>
+      <img src="img/logo.jpg" alt="youtube img">
+    </div>
+    <div class="col-md-6  search-bar">
+      <input type="text" name="text" placeholder="Search">
+      <button for="text"><i class="fas fa-search"></i></button>
+    </div>
+    <div class="col-md-3  news">
+      <a href="">
+        <i class="fas fa-video"></i>
+      </a>
+      <a href="">
+        <i class="fas fa-th"></i>
+      </a>
+      <a href="">
+        <i class="fas fa-bell"></i>
+      </a>
+      <a href="">
+        <i class="fas fa-user-circle"></i>
+      </a>
+    </div>
+  </header>
+  <div>
+    <div class="container">
+      <section class="main">
+        <div class="video-principal">
+          <iframe width="727" height="409" src="https://www.youtube.com/embed/g_0QtAwkVh8" frameborder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="video-info">
+          <div id="info">
+            <h1>Is The New 2020 Ducati Superleggera V4 Worth €100,000?</h1>
+          </div>
+          <div id="info-1">
+            <div id="info-text">
+              <p class="info-p">480,905 views . 5 March 2020 </p>
+            </div>
+            <div id="top-buttons">
+              <a href="">
+                <i class="fas fa-thumbs-up"></i>
+              </a>
+              <p class="info-p">5560</p>
+              <a href="">
+                <i class="fas fa-thumbs-down"></i>
+              </a>
+              <p class="info-p">197</p>
+              <a href="">
+                <i class="fas fa-share"></i>
```

Comment:
The share icon should be a button:
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))